### ISSUE TITLE
_firstRendered triggers after _shouldRender returns false.

### ISSUE THREAD

**Issue by Tanbouz** [Polymer/lit-element#131]
### Description
`_firstRendered` triggers after ` _shouldRender` returns false.

#### Live Demo
https://stackblitz.com/edit/lit-element-example-5rtmtq

#### Steps to Reproduce
As in live demo
```
import {LitElement, html} from '@polymer/lit-element';

class MyElement extends LitElement {

  static get properties() { return { mood: String }}

  _render({mood}) {
    return html`<style> .mood { color: green; } </style>
      Web Components are <span class="mood">${mood}</span>!`;
  }
  _shouldRender(props, changedProps, prevProps){
    return false;
  }

  _firstRendered() {
    console.log("First render!");
  }
}

customElements.define('my-element', MyElement);
```

#### Expected Results
`_firstRendered` is not called when `_shouldRender` returns false. 

At least from what I understood from the description of the callbacks in the readme or what made sense to me from the naming.

#### Actual Results
`console.log("First render!");` prints to console

### Browsers Affected
- [x] Chrome
- [x] Firefox
- [?] Edge
- [?] Safari 11
- [?] Safari 10
- [?] IE 11

### Versions

- lit-element: v0.5.2
- webcomponents: v2.0.3

**Comment by bennypowers** [comment]
See also https://github.com/Polymer/lit-element/issues/116

**Comment by sorvell** [comment]
This has been fixed in master and the `dev` release and will be included in the `0.6` release.
